presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/AbstractTestHiveFileSystemAbfs.java

```diff
@@ -106,12 +94,13 @@ private void ensureTableExists(SchemaTableName table, String tableDirectoryName,
         }
     }
 
+    /** Given an empty {@code HiveAzureConfig}, return a configured instance. */
```

Comment:
That's kind of an unusual contract.  Why do we need an empty `HiveAzureConfig` as an argument?  The method could just instantiate one.